packages/astro/src/container/index.ts

```diff
@@ -329,10 +329,10 @@ export class experimental_AstroContainer {
 	private static async createFromManifest(
 		manifest: SSRManifest
 	): Promise<experimental_AstroContainer> {
-		const astroConfig = await validateConfig(ASTRO_CONFIG_DEFAULTS, process.cwd(), 'container');
+		// const astroConfig = await validateConfig(ASTRO_CONFIG_DEFAULTS, process.cwd(), 'container');
```

Comment:
if this code isn't needed anymore than we should remove it rather than comment it out

Reply:
Good catch, I was looking for it. I have to re-enable the code